README.md

```diff
@@ -35,6 +35,44 @@ bun i -D svelte-geolocation
 yarn add -D svelte-geolocation
 ```
 
+### Known Issues
```

Comment:
Note this is in the `Installation` section of the README. Should it go in the `Usage` section instead, I wonder?